webui/src/theme-provider.tsx

```diff
@@ -0,0 +1,66 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+import * as React from 'react';
+
+import { ThemeProvider as MuiThemeProvider } from '@material-ui/styles';
+import { createMuiTheme } from '@material-ui/core';
+import { Main } from './main';
+import { PageSettings } from '../src/page-settings';
+import { ExtensionRegistryService } from '../src/extension-registry-service';
+
+const ThemeProvider = (props: ThemeProvider.Props) => {
```

Comment:
The name collision with the MUI component wasn't clear to me. Maybe we should rename this to ThemeController or something like that?

Reply:
yeah! Thanks, ThemeController seems more appropriate.